modules/nf-core/agat/spaddintrons/meta.yml

```diff
@@ -0,0 +1,40 @@
+---
+# yaml-language-server: $schema=https://raw.githubusercontent.com/nf-core/modules/master/modules/meta-schema.json
```

Comment:
```suggestion
```

Reply:
thanks, i assumed this was some necessary markup.